src/components/TextareaField/TextareaField.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Textarea, Label } from './styles';
+
+const TextareaField = ({ label, placeholder, onChange }) => {
+  const handleInputChange = (e) => {
+    onChange(e.target.value);
+  };
+
+  return (
+    <div>
+      {label && <Label htmlFor="note">{label}</Label>}
+      <Textarea id="note" name="note" onChange={handleInputChange} placeholder={placeholder} />
```

Comment:
Where is a `value` prop?
https://reactjs.org/docs/forms.html#controlled-components